sr_demos/scripts/sr_demos/demo.py

```diff
@@ -441,11 +475,13 @@ def rock_paper_scissors(self):
             self.commander.move_to_named_target("fingers_pack_thumb_open")
         self.commander.move_to_named_target("open")
         rospy.loginfo("Rock, Paper, Scissors demo completed")
+        self._enable_touch_detection(True)
 
     def grasp_demo(self):
```

Comment:
No type hinting?

Reply:
fixed in https://github.com/shadow-robot/sr_interface/pull/823/commits/4d899740e14c6a37b20146403ea06d3b82a837c8